Slam/Queue/APIManagedQueue.swift

```diff
@@ -9,10 +9,23 @@ public class APIManagedQueue: QueueManager {
         self.queueView = queueView
     }
 
+    public init() {
+        self.api = SlamAPI(httpClient: AsynchronousHTTPClient())
+        self.queueView = nil
+    }
+
     public func loadQueue(onSuccess: () -> Void) {
         api.getQueue({(matches: [Match]) in
-            self.queueView.showMatches(matches)
+            self.queueView!.showMatches(matches)
```

Comment:
Using `!` outside of tests is a bit dangerous, since it would cause a runtime error. If `queueView` remains as an Optional, this should probably be a `?`.